prover/Cargo.toml

```diff
@@ -22,7 +22,8 @@ std = ["air/std", "processor/std", "winter-prover/std"]
 air = { package = "miden-air", path = "../air", version = "0.8", default-features = false }
 processor = { package = "miden-processor", path = "../processor", version = "0.8", default-features = false }
 tracing = { version = "0.1", default-features = false, features = ["attributes"] }
-winter-prover = { package = "winter-prover", version = "0.8", default-features = false }
+winter-air = { package = "winter-air", version = "0.9", default-features = false }
```

Comment:
nit: rather than importing `winter-air` here, it might be better to re-export w/e is needed here from `miden-air` and then import it here from `miden-air`.

Reply:
done